modules/publish/Actions.php

```diff
@@ -101,7 +101,7 @@ public function preExecute(framework\Request $request, $action)
                     }
                     elseif ($request->hasParameter('parent_article_name'))
                     {
-                        $this->article->setParentArticle(tables\Articles::getTable()->getArticleByName($request['parent_article_name']));
+                        $this->article->setParentArticle(entities\tables\Articles::getTable()->getArticleByName($request['parent_article_name']));
```

Comment:
The whole namespace declaration shouldn't be needed. Adding a use statement for Articles should solve this:
```php
use thebuggenie\modules\publish\entities\tables\Articles;
```
I'd even like to propose to capitalize/camelcase all use statements, so it would become:
```php
use TheBugGenie\Modules\Publish\Entities\Tables\Articles;
```